### DISCUSSION TITLE
Hyperkit driver is not supported on Apple M1 computers

### DISCUSSION THREAD

**Discussion by victorchapman** [apache/camel-karavan#888] [Q&A] [answered]
<img width="459" alt="Screenshot 2023-09-11 at 1 05 42 PM" src="https://github.com/apache/camel-karavan/assets/65624008/52abde7e-0483-4c64-9345-64630d6d137b">

Is it possible to still run the web version of Karavan on minikube?

**Comment by victorchapman** [comment]
Does qemu work?

**Comment by mgubaidullin** [comment, marked as answer]
On Apple Mx I use --driver=docker
